java-components/build-recipes-database/src/test/resources/test-recipes/build-info/github.com/quarkusio/quarkus/_version/1.9999/build.yaml

```diff
@@ -1,3 +1,5 @@
 ---
 additionalArgs:
   - "-Dquarkus1=true"
+excludes:
```

Comment:
Sorry I did not pick up on this earlier, but I think we need a more descriptive name for this (and all the downstream stuff like CRDs etc). 

'excludes' does not really mean much, maybe 'allowedDifferences'?